stock_inventory_barcode/__init__.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    OpenERP, Open Source Management Solution
+#    Copyright (C) 2010-2013 OpenERP s.a. (<http://openerp.com>).
```

Comment:
Don't put Copyrights OpenERP s.a. when it's not the case (and I don't think it is the case here !